### PR TITLE
9.0 mig stock picking back2draft

### PR DESCRIPTION
**Pull request by nicomacr**
MIG v9 stock picking back2draft

### CODE REVIEW

File: stock_picking_back2draft/README.rst
Comment (pedrobaeza) [inline stock_picking_back2draft/README.rst]: s/8.0/9.0

**Comment by jjscarafia** [comment]
:+1: tested on runbot

File: stock_picking_back2draft/models/stock.py
Comment (pedrobaeza) [inline stock_picking_back2draft/models/stock.py]: No need to do the check 2 times.

**Review by Emiq2 — APPROVED** [review]

**Comment by jjscarafia** [comment]
Any news with this? One more reviewer please?

**Comment by pedrobaeza** [comment]
Please squash together all the "OCA Transbot updated translations from Transifex" and the commit " 	remove en.po that was erroneously created by transbot" that it's also for translations and I'll merge.

**Comment by JordiBForgeFlow** [comment]
@lreficent can you review?
@nicomacr Can you squash the commits, as Pedro suggested? See here instructions on how to do that: https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests#mergesquash-the-commits-generated-by-transifex

**Comment by florian-dacosta** [comment]
Hello,
I think this module is too simple and works only for a few cases, as it does not handle procurements.
It may have unexpected behavior for user in case of chained pickings for example no?

If we have a 2 steps delivery with manufactured product for instance, if you put the delivery order back to draft, it will recreate procurements, the previous chained picking, the manufactured order... And I guess the user just expects to put the Delivery order the same way as before because he canceled it by mistake! He may has already done the manufactured order and so on.

So I guess it works for very simple case only. I would not want to block this PR as in version 8, the funcionalities are the same, but I think it would be good to put a note in the README file to explain which cases are handled or not.

**Comment by JordiBForgeFlow** [comment]
@nicomacr Are you planning to follow up on this one? Otherwise we will continue the work

**Comment by lreficent** [comment]
Forced rebuilt on runbot and still not working.

**Review by LoisRForgeFlow — APPROVED** [review]
Functional and technical review. LGTM

**Comment by LoisRForgeFlow** [comment]
However, despite this works, I think what @florian-dacosta pointed is important and is not covered by this module.

**Comment by florian-dacosta** [comment]
Yes, if you want to merge this module the way it is now, no problem, but please, add a comment in the README file to inform the users that it works only for simple picking (Not chained picking).

Or maybe raise a UserError if a user try to put a picking to draft state and at least one move has the field `procure_method` to `make_to_order` value? Because this mean it will re-create a procurement, and I don't think this is a wanted behavior.

**Comment by Cedric-Pigeon** [comment]
@nicomacr Could integrate @florian-dacosta  remarks in order to merge this PR ?

**Comment by rousseldenis** [comment]
@nicomacr

**Comment by rousseldenis** [comment]
@nicomacr Could you squash all migration commits ?

**Comment by rousseldenis** [comment]
@pedrobaeza Could you update your review?

**Comment by pedrobaeza** [comment]
I still see a lot of noisy commits in the PR:
seleccion_018

**Comment by rousseldenis** [comment]
@pedrobaeza See my remark above. I was talking about README and stock.py files. I can assign needs fixing for commits.

**Comment by pedrobaeza** [comment]
Yeah, but I want that commits to be squashed.

File: stock_picking_back2draft/__openerp__.py
Comment (Nikul-Chaudhary) [inline stock_picking_back2draft/__openerp__.py]: @nicomacr URL should be https://github.com/OCA/stock-logistics-workflow

**Review by Nikul-Chaudhary — CHANGES_REQUESTED** [review]
Minor change

File: stock_picking_back2draft/__openerp__.py
Comment (nicomacr) [reply stock_picking_back2draft/__openerp__.py]: Done!

**Review by rousseldenis — APPROVED** [review]
Code review

**Comment by nicomacr** [comment]
@pedrobaeza done the squash in this PR.

**Comment by rousseldenis** [comment]
@Nikul-Chaudhary

**Comment by pedrobaeza** [comment]
Thanks.

**Comment by rousseldenis** [comment]
@pedrobaeza Even if the modification was done, I'd prefer let time to reviewers that have made remarks to update their review and make PR's - as possible- with all green reviews before merge.

Be moderate (not too fast, not too slow) :smiley:

**Comment by pedrobaeza** [comment]
Your points are not correct:

- Everything (runbot, Travis and codecov) is green
- There are not pending remarks
- Count my merge as second review

**Comment by rousseldenis** [comment]
Missing Nik approval

**Comment by pedrobaeza** [comment]
> Missing Nik approval

No, he hasn't put "Request changes" review and I have checked that his comment has been attended.

**Pull request merged**
